ironfish/src/rpc/routes/accounts/getTransactions.ts

```diff
@@ -61,19 +62,26 @@ router.register<typeof GetAccountTransactionsRequestSchema, GetAccountTransactio
 
     if (request.data.hash) {
       const hash = Buffer.from(request.data.hash, 'hex')
-      const transaction = await account.getTransactionByUnsignedHash(hash)
 
-      if (transaction) {
-        await streamTransaction(request, node, account, transaction)
-      }
+      await node.accounts.db.database.transaction(async (tx) => {
```

Comment:
I'm not sure we need this transaction exactly. It looks like it'll be fine without it